exporter/src/main.rs

```diff
@@ -112,7 +112,6 @@ fn take_screenshot(
         .with_renderer(
             WgpuRenderBackend::new(descriptors, target).map_err(|e| anyhow!(e.to_string()))?,
         )
-        .with_software_video()
```

Comment:
Shouldn't the exporter (optionally) be able to display video?

Reply:
Yes probably! Right now it doesn't, this line is basically a no-op, it was misleading :D

Didn't want to change behaviours in this PR.

Reply:
Ah, I see, previously `exporter` didn't enable any of the video features in core so all video encoders would be excluded!